lua/astrocommunity/motion/harp-nvim/README.md

```diff
@@ -0,0 +1,15 @@
+# harp.nvim
+
+**Repositories:** 
+
+Harp: https://github.com/Axlefublr/harp (**Required to use the plugin**)
```

Comment:
Add this under dependencies. See other plugins.

Reply:
It's still a dependency of the plugin itself.

Reply:
Do you have an example? The only other plugin I had seen that had an external binary as a dependency didn't have it listed in their readme, just on the plugins repo itself (this is distant specifically, haven't seen other plugins that had it like I said)

Reply:
A quick ripgrep gave me this: 
<img width="876" alt="image" src="https://github.com/AstroNvim/astrocommunity/assets/48220549/c41fb0a0-c92e-43e4-93bd-1209e4391586">

:)

Reply:
Thanks!